### PR TITLE
refactor: Move Badge to Antdesign

### PR DESCRIPTION
**Pull request by geido**
### SUMMARY
Move the Badge component from react-bootstrap to Antdesign and add the ability to change text color.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
Before:
badge-before
After:
Screen Shot 2020-12-04 at 17 16 18

### TEST PLAN
1. Log into Superset
2. Navigate to Datasets
3. Edit a dataset
4. Look for the badges for METRICS, COLUMNS, and CALCULATED COLUMNS

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by geido** [comment]
@junlincc @rusackas

File: superset-frontend/stylesheets/less/cosmo/variables.less [hunk 1]
```diff
@@ -751,24 +751,6 @@
...
-
-// ** Badge text color in active nav link
-@badge-active-color: @link-color;
-// ** Badge background color in active nav link
-@badge-active-bg: @lightest; // superset-var
-
-@badge-font-weight: bold;
-@badge-line-height: 1;
-@badge-border-radius: 10px;
-
```
Comment (rusackas) [inline superset-frontend/stylesheets/less/cosmo/variables.less:771]: OMG! So happy to see less LESS.

**Comment by rusackas** [comment]
Code LGTM, and I'm ready to approve on that, but the text in the badges looks just _slightly_ off-center, to the right. Have you looked at what might be causing that?

File: superset-frontend/stylesheets/less/cosmo/variables.less (hunk 1 above)
Comment (mistercrunch) [reply superset-frontend/stylesheets/less/cosmo/variables.less:771]: I think it's ok to remove this, but generally I'd keep the theme intact for where FAB is used (the goal is to remove all of the FAB templates, but there are still some CRUD pages that are FAB-powered, and if they have a bootstrap badge they'll won't be themed)

Comment (rusackas) [reply superset-frontend/stylesheets/less/cosmo/variables.less:771]: Ahh, good catch... I got too excited.

Comment (geido) [reply superset-frontend/stylesheets/less/cosmo/variables.less]: Restored. Thanks

**Comment by geido** [comment]
> Code LGTM, and I'm ready to approve on that, but the text in the badges looks just _slightly_ off-center, to the right. Have you looked at what might be causing that? The 1 looks good in the screenshot, the 4 looks a bit off (might be the optical weight of the shape of the number), but the 0 should look centered, I would think.

@rusackas Thanks for noticing. Should look better now

**Review by rusackas — APPROVED** [review]
LGTM! Thanks for the touchups!

**Pull request merged**